### ISSUE TITLE
Very preterm

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7270]
**Preferred term label:**
Very preterm

**Synonyms**

**Definition (free text, please give PubMed ID)**
Very preterm is defined as babies born alive during the period after 28 weeks to before 32 weeks of gestation (PMID: 27743648).

**Parent term (use hpo.jax.org/app)**
Premature birth

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
added
